rules/windows/builtin/win_powershell_script_installed_as_service.yml

```diff
@@ -0,0 +1,30 @@
+title: PowerShell Scripts Installed as Services
+id: a2e5019d-a658-4c6a-92bf-7197b54e2cae
+description: Detects powershell script installed as a Service
+status: experimental
+author: oscd.community, Natalia Shornikova
+date: 2020/10/06
+references:
+    - https://speakerdeck.com/heirhabarov/hunting-for-powershell-abuse
+tag:
+    - attack.execution
+    - attack.t1569.002
+logsource:
+    product: windows
+detection:
+    selection1:
+      EventID:
+        - 7045
+        - 4697
```

Comment:
since you are using event ids from multiple logsources, you need to reconstruct your rule utilizing [Rule Collections](https://github.com/Neo23x0/sigma/wiki/Specification#rule-collections). You can use [this](https://github.com/Neo23x0/sigma/blob/master/rules/windows/builtin/win_mal_creddumper.yml) rule as an example (: